rest_framework_filters/filterset.py

```diff
@@ -39,6 +39,16 @@ def __new__(cls, name, bases, attrs):
         new_class = super(FilterSetMetaclass, cls).__new__(cls, name, bases, attrs)
         fix_filter_field = _get_fix_filter_field(new_class)
 
+        # order_by is not compatible.
+        if new_class._meta.order_by:
+            new_class._meta.order_by = False
+            warnings.warn(
+                'order_by is no longer supported. Use '
+                'rest_framework.filters.OrderingFilter instead. See: '
+                'https://github.com/philipn/django-rest-framework-filters/issues/62',
```

Comment:
You whoops - pretty sure I meant to reference 72.

Thanks, added a note to the issue.